glmmTMB/tests/testthat/test-families.R

```diff
@@ -33,13 +33,6 @@ test_that("binomial", {
     expect_equal( logLik(mod4)     , logLik(mod5) )
     expect_equal( fixef(mod4)$cond , fixef(mod5)$cond )
 
-    ## Now with extra weights
-    dd$w <- 2
-    mod6 <- glmmTMB(cbind(success,failure)~1,family=binomial,data=dd,weights=w)
-    mod7 <- glmmTMB(prop~1,weights=size*w,family=binomial,data=dd)
```

Comment:
@mebrooks  Could we keep this test ? We still allow prop/weights right ?

Reply:
In that case I'd suggest modifying the test to check the likelihood of each of the two fits against `glm`.

Reply:
@kaskr I put back the check of the `fixef` but removed the `logLik` check.

Reply:
@kaskr Sorry, I didn't see the suggestion to compare to `glm`. I just made *another* commit that does this.